provisioning/service/tests/Config/TpmAttestationTests.cs

```diff
@@ -41,16 +42,20 @@ public void TpmAttestation_ThrowsOnEnptyEndorsementKey()
 
         [TestMethod]
         [TestCategory("DevService")]
-        public void TpmAttestation_ThrowsOnNonBase64EndorsementKey()
+        public void TpmAttestation_SucceedOnNonBase64EndorsementKey()
```

Comment:
Tests describe the requirements, they are not testing things that are _not_ required.